compiler/src/iree/compiler/Dialect/HAL/Target/LLVM/LLVMCPUTarget.cpp

```diff
@@ -843,21 +843,9 @@ void registerLLVMCPUTargetBackends(
     return std::make_shared<LLVMCPUTargetBackend>(queryOptions());
   };
 
-  // Preferred name.
+  // #hal.device.target<"llvm-cpu", ...
   // #hal.executable.target<"llvm-cpu", ...
-  static TargetBackendRegistration registration0("llvm-cpu", backendFactory);
-
-  // Abbreviated names.
-  // #hal.device.target<"cpu", ...
-  static TargetBackendRegistration registration1("cpu", backendFactory);
-  // #hal.executable.target<"llvm", ...
-  static TargetBackendRegistration registration2("llvm", backendFactory);
```

Comment:
Seeing failures relating to target names and device IDs: https://github.com/iree-org/iree/blob/3bc74ec7c66cc9ac77131b99ae5253e55d67725c/compiler/src/iree/compiler/Dialect/HAL/Target/LLVM/LLVMCPUTarget.cpp#L147-L149

`/home/runner/actions-runner/_work/iree/iree/samples/static_library/simple_mul.mlir:0:0: error: unregistered target backend "cpu"; ensure it is linked in to the compiler (available = [ 'cuda' 'llvm-cpu' 'metal' 'metal-spirv' 'vmvx' 'vulkan' 'vulkan-spirv' ])`

just changing those to `llvm-cpu` isn't enough, so I'm debugging